website/source/layouts/docs.erb

```diff
@@ -104,6 +104,15 @@
 					<a href="/docs/http/index.html">API & Libraries</a>
 				</li>
 
+				<li<%= sidebar_current("docs-guides") %>>
+					<a href="/docs/guides/index.html">Guides</a>
+					<ul class="nav">
+						<li<%= sidebar_current("docs-guides-generate-root") %>>
+							<a href="/docs/guides/generate-root.html">Generate Root</a>
```

Comment:
I think in the sidebar it'd be better to say "Root Token Generation" here. "Generate Root" is the Vault command but for a newbie it's already slightly inside-baseball. I'll change that and merge it, but if you have strong feelings here, feel free to let me know.